packages/app-project/src/screens/ProjectAboutPage/components/TeamMember.js

```diff
@@ -66,27 +68,27 @@ const TeamMember = ({ user, router }) => {
   const placeholderAvatar = `${assetPrefix}/simple-avatar.png`
 
   return (
-    <StyledTeamMember as="li">
+    <StyledTeamMember as='li'>
       <StyledAvatar>
         {!user.avatar_src ? (
           <Image 
-            alt="Placeholder Avatar"
-            fit="cover" 
+            alt='Placeholder Avatar'
+            fit='cover' 
             src={placeholderAvatar} 
           />
         ) : (
-          <Image alt={user.display_name} fit="cover" src={user.avatar_src} />
+          <Image alt={user.display_name} fit='cover' src={user.avatar_src} />
         )}
       </StyledAvatar>
-      <Box flex direction="column">
-        <StyledDisplayName>{user.display_name}</StyledDisplayName>
+      <Box flex direction='column'>
+        <StyledDisplayName color={{ light: 'neutral-7' }}>{user.display_name}</StyledDisplayName>
```

Comment:
Similar comment here:

```suggestion
        <StyledDisplayName color={{ light: 'neutral-7', dark: '' }}>{user.display_name}</StyledDisplayName>
```